tests/platform_tests/project_BIDS_multi_session_work-dir_succeeds.py

```diff
@@ -0,0 +1,131 @@
+#! /usr/bin/env python3
+"""Run bids-fmriprep on project "BIDS_multi_session"
+
+    This script was created to run Job ID 61aa7f687a4ab255b14ab884
+    In project "bids-apps/BIDS_multi_session"
+    On Flywheel Instance https://ga.ce.flywheel.io/api
+"""
+
+import argparse
+import os
+from datetime import datetime
+
+import flywheel
+
+input_files = {
+    "fs-subjects-dir": {
+        "container_path": "bids-apps/BIDS_multi_session/sub-TOME3024/ses-Session2/analyses/bids-freesurfer "
+        "10-13-2021 15:03:53 SDK launched",
+        "location_name": "bids-freesurfer_ses_Session2_61672d99ed5fcc2ee72d8488.zip",
+    },
+    "work-dir": {
+        "container_path": "analysis bids-apps/BIDS_multi_session",
+        "location_name": "bids-fmriprep_work_BIDS_multi_session_61a807007a4ab255b14aaf78.zip",
+    },
+}
+
+
+def main(fw):
+
+    gear = fw.lookup("gears/bids-fmriprep")
+    print("gear.gear.version in original job was = 1.2.2_20.2.6_rc6")
+    print(f"gear.gear.version now = {gear.gear.version}")
+    print("destination_id = 6151e90263e0a8412b4ec863")
+    print("destination type is: project")
+    destination = fw.lookup("bids-apps/BIDS_multi_session")
+
+    inputs = dict()
+    for key, val in input_files.items():
+        if val["container_path"][:8] == "analysis":
+            path = val["container_path"][9:]
+            parent_of_analysis = fw.lookup(path)
+            # find analysis that has the right file
+            analyses = parent_of_analysis.reload().analyses
+            for analysis in analyses:
+                for file in analysis.files:
+                    if file.name == val["location_name"]:
+                        container = analysis
+        else:
+            container = fw.lookup(val["container_path"])
+        inputs[key] = container.get_file(val["location_name"])
+
+    config = {
```

Comment:
Is there a reason config isn't in conftest.py, so you didn't have to keep defining it?

Reply:
These tests are not run by pytest because they don't start with "test_".  They are run by hand.  Someday they will be run as part of a system of end-to-end tests.  For now they are just an easy way to run a test on the platform after the pytests pass.